nifi-nar-bundles/nifi-media-bundle/nifi-media-processors/src/test/java/org/apache/nifi/processors/image/ExtractImageMetadataTest.java

```diff
@@ -37,7 +37,7 @@
     private static String BMP_HEADER = "BMP Header.";
     private static String JPEG_HEADER = "JPEG.";
     private static String GIF_HEADER = "GIF Header.";
-    private static String PNG_HEADER = "PNG.";
+    private static String PNG_HEADER = "PNG-";
```

Comment:
Yes, [Drew Noakes Metadata Extractor](https://drewnoakes.com/code/exif/) appears to have changed.  The Tika parsers can't work with the older version, and they don't coexist well with the same nar bundle.

The only option I see for these to co-exist without this change is to revert the nifi-image-bundle back to what it was and create nifi-media-bundle as a whole new bundle, but that seems like it will be confusing down the road.  What do you think?